theme/ting_agency.theme.inc

```diff
@@ -92,7 +92,7 @@ function get_ting_agency_tools($branch) {
   $items = array();
   if (!empty($links)) {
     foreach ($links as $name => $link) {
-      $item['data'] = l($name, $link, array());
+      $item['data'] = l($name, $link, array('attributes' => array('target' => '_blank')));
```

Comment:
Flueknepperi bare for at skrive noget - linjen er mere end 80 tegn. Bryd den op :-)